recipe/meta.yaml

```diff
@@ -12,7 +12,14 @@ source:
     - patches/0001-Instructions-for-building-a-functional-Flang-cross-c.patch
     - patches/0002-Minimal-WASM-support-for-flang.patch
     - patches/0003-Specialize-Flang-to-target-WASM.patch
-    - patches/0004-Patches-to-build-flang-runtime-with-emcc.patch
+    # see https://github.com/llvm/llvm-project/pull/99465
+    - patches/0004-Fix-implicit-conversion-warning-when-targeting-32bit.patch
+    # see https://github.com/llvm/llvm-project/pull/99822
+    - patches/0005-Explicitly-convert-shift-value-to-SubscriptValue.patch
+    # see https://github.com/llvm/llvm-project/pull/101242
+    - patches/0006-Handle-missing-definitions-in-cfenv.patch
```

Comment:
So https://github.com/llvm/llvm-project/commit/6db2465ce74141f378a290cfa3b56eb69dc379cb already landed before 19.1.0.rc1 (so it needs to be dropped), while the commits for patch 0006 don't apply on the 19.x branch (due to https://github.com/llvm/llvm-project/commit/4cdc19b84cf48189e40915178d60991f6fc469b2). I'm tentatively adding that patch as a backport as well for now, even though it's larger than I'd normally be comfortable with.